examples/convert_nirs_to_bids.py

```diff
@@ -0,0 +1,247 @@
+"""
+====================================
+13. Convert NIRS data to BIDS format
+====================================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of NIRS
+data. Specifically, we will follow these steps:
+
+1. Download some NIRS data from a github repository
+
+2. Load the data, extract information, and save it in a new BIDS directory.
+
+3. Check the result and compare it with the standard.
+
+4. Cite ``mne-bids``.
+
+.. currentmodule:: mne_bids
+
+.. _BrainVision format: https://www.brainproducts.com/productdetails.php?id=21&tab=5
+.. _CapTrak: https://www.fieldtriptoolbox.org/faq/coordsys/#details-of-the-captrak-coordinate-system
+
+"""  # noqa: E501
+
+# Authors: Robert Luke <code@robertluke.net>
+#
+# License: BSD-3-Clause
+
+# %%
+# We are importing everything we need for this example:
+import os.path as op
+import shutil
+import tempfile
+
+import pooch
+import mne
+
+from mne_bids import write_raw_bids, BIDSPath, print_dir_tree
+from mne_bids.stats import count_events
+
+# %%
+# Download the data
+# -----------------
+#
+# First, we need some data to work with. We will use the
+# `Finger Tapping Dataset <https://github.com/rob-luke/BIDS-NIRS-Tapping>`_
+# available on GitHub.
+#
+# The data consists of 109 volunteers performing 14 experimental runs each.
+# For each subject, there were two baseline tasks (i) eyes open, (ii) eyes
+# closed, as well as four different motor imagery tasks.
+#
+# In this example, we will download the data for a single subject doing the
+# baseline task "eyes closed" and format it to the Brain Imaging Data Structure
+# (`BIDS <https://bids.neuroimaging.io/>`_).
+#
+
+# Create a temporary directory to store the data in
+data_dir = tempfile.mkdtemp()
```

Comment:
Maybe it's easier for this example to depend on mne-nirs? Then you can just do `mne_nirs.datasets.fnirs_motor_group.data_path()`.

It only adds a dependency (plus its deps, like `nilearn`) to the doc build, not the library, so this seems preferable to me.

Reply:
Sure, will do now and we can revert if required.

Reply:
Shouldn't we add MNE-NIRS to the library deps anyway?

Reply:
Ah no, scratch that :)